about.html

```diff
@@ -1,67 +1,110 @@
-      </div>
     </header>
 
-    <div class="about-page-container"> <!-- Main Container -->
-
-    </div> <!-- End of Main Container -->
-
-    <footer>
-      <div class="footer-link-container"> <!-- Footer Links -->
-        <a class="footer-link" href="index.html">Home</a>
-        <a class="footer-link" href="features.html">Features</a>
-        <a class="footer-link" href="about.html">About</a>
-        <a class="footer-link" href="contact.html">Contact</a>
-      </div> <!-- End of Footer Links -->
-
-      <div class="footer-icon-container"> <!-- Footer Icon -->
-        <div>
-          <a href="index.html" class="icon-link"><i class="fas fa-dumbbell footer-img"></i></a>
+    <div class="about">
+      <h2>Meet Our Amazing Team</h2>
+      <div class="team">
+        <div class="card">
+          <div class="img-container">
+            <img src="images/team/amanda.jpeg" alt="Amanda Hogan" />
+          </div>
+          <h4>Amanda Hogan</h4>
+          <p>Landing Page</p>
+          <a href="https://github.com/anhogan">GitHub</a>
         </div>
-        <p class="footer-copyright">Copyright 2020</p>
-      </div> <!-- End of Footer Icon -->
-    </footer>
+        <div class="card">
+          <div class="img-container">
+            <img src="images/team/austin.jpeg" alt="Austin Walela" />
+          </div>
+          <h4>Austin Walela</h4>
+          <p>Landing Page</p>
+          <a href="https://github.com/lqtm">GitHub</a>
+        </div>
+        <div class="card">
+          <div class="img-container">
+            <img src="images/team/david.png" alt="David Tauraso" />
+          </div>
+          <h4>David Tauraso</h4>
+          <p>Frontend Engineer</p>
+          <a href="https://github.com/anhogan">GitHub</a>
+        </div>
+        <div class="card">
+          <div class="img-container">
+            <img src="images/team/edwin.jpeg" alt="Edwin Deronvil" />
+          </div>
+          <h4>Edwin Deronvil</h4>
+          <p>Frontend Engineer</p>
+          <a href="https://github.com/anhogan">GitHub</a>
+        </div>
+        <div class="card">
+          <div class="img-container">
+            <img src="images/team/susan.jpeg" alt="Susan Trimnal" />
+          </div>
+          <h4>Susan Leigh Trimnal</h4>
+          <p>Backend Engineer</p>
+          <a href="https://github.com/anhogan">GitHub</a>
+        </div>
+        <div class="card">
+          <div class="img-container">
+            <img src="images/team/zachary.jpeg" alt="Zach Bumpous" />
+          </div>
+          <h4>Zach Bumpous</h4>
+          <p>Backend Engineer</p>
+          <a href="https://github.com/anhogan">GitHub</a>
+        </div>
+        <div class="card">
+          <div class="img-container">
+            <img src="images/team/brandy.jpeg" alt="Brandy Becker" />
+          </div>
+          <h4>Brandy Becker</h4>
+          <p>Project Lead</p>
+          <a href="https://github.com/anhogan">GitHub</a>
```

Comment:
I would change all of these to each person's GitHub link - they should all be listed here, or in one of the other 2 repos